<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PHDLink</class>
 <widget class="QDialog" name="PHDLink">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PHD Link</string>
  </property>
  <widget class="QGroupBox" name="grpBox_durDownload">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>During Downlaod</string>
   </property>
   <widget class="QRadioButton" name="radioBtn_active">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Active</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtn_paused">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Paused</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pBtn_connect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_connInfo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connection OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpBox_ditherSetting">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Dither Setting</string>
   </property>
   <widget class="QComboBox" name="comboBox_ditherLevel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No Dither</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Small Dither</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medium Dither</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>High Dither</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Very High Dither</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Extreme Dither</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_settleRange">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Settle at &lt; 0.1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Settle at &lt; 0.2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Settle at &lt; 0.3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Settle at &lt; 0.4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Settle at &lt; 0.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Settle at &lt; 0.6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Settle at &lt; 0.7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Settle at &lt; 0.8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Settle at &lt; 0.9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Settle at &lt; 1.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Settle at &lt; 1.1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Settle at &lt; 1.2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Settle at &lt; 1.3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Settle at &lt; 1.4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Settle at &lt; 1.5</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_ditherLevel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dither Level</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_settleRange">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Settle Range</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Test ControlDither</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
